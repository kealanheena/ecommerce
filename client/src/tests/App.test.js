import React from 'react';
import { render } from '@testing-library/react';
//import App from '../App';

test('matches snapshot', () => {
  //const renderedApp = render(<App />);
  expect(true).toBe(true);
});
