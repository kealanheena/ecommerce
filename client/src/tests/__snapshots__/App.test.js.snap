// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="App"
      >
        <nav
          class="navbar navbar-expand navbar-dark bg-dark"
        >
          <a
            class="navbar-brand"
            href="/"
          >
            Ecommerce
          </a>
          <div
            class="mr-auto navbar-nav"
          >
            <a
              class="nav-link"
              href="/"
            >
              Home
            </a>
            <a
              class="nav-link"
              href="/products"
            >
              products
            </a>
            <a
              class="nav-link"
              href="/products/:id"
            >
              pdp
            </a>
          </div>
        </nav>
        <div
          class="homePage"
        >
          <h1>
            Home page
          </h1>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="App"
    >
      <nav
        class="navbar navbar-expand navbar-dark bg-dark"
      >
        <a
          class="navbar-brand"
          href="/"
        >
          Ecommerce
        </a>
        <div
          class="mr-auto navbar-nav"
        >
          <a
            class="nav-link"
            href="/"
          >
            Home
          </a>
          <a
            class="nav-link"
            href="/products"
          >
            products
          </a>
          <a
            class="nav-link"
            href="/products/:id"
          >
            pdp
          </a>
        </div>
      </nav>
      <div
        class="homePage"
      >
        <h1>
          Home page
        </h1>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
